addons/user-status/addon.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "Show user statuses from external sites",
+  "description": "Show ocular.jeffalo.net and aviate.scratchers.tech statuses on profiles and forum posts.",
+  "tags": ["community", "forums", "profiles"],
+  "versionAdded": "1.34.0",
+  "userscripts": [
+    {
+      "url": "discuss.js",
+      "matches": ["topics"],
+      "if": {
+        "settings": { "discuss": true }
+      }
+    },
+    {
+      "url": "profile.js",
+      "matches": ["https://scratch.mit.edu/users/*/"],
+      "if": {
+        "settings": { "profile": true }
+      }
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["topics", "https://scratch.mit.edu/users/*/"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Show Ocular statuses",
+      "id": "ocular",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Show Aviate statuses",
+      "id": "aviate",
+      "type": "boolean",
+      "default": true
```

Comment:
`"default": true` on Aviate probably won't make it

Reply:
Fair

```suggestion
      "default": false
```